### PR TITLE
cron.php calls occ system:cron as fallback

### PR DESCRIPTION
**Pull request by DeepDiver1975**
## Description
cron.php calls the occ system:cron command as fallback

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Database schema changes (next release will require increase of minor version instead of patch)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Technical debt
- [ ] Tests only (no source changes)

## Checklist:

- [x] Code changes
- [ ] Unit tests added
- [ ] Acceptance tests added
- [ ] Documentation ticket raised: <link> 

## Open tasks:

- [ ] Backport (if applicable set "backport-request" label and remove when the backport was done)

### CODE REVIEW

File: cron.php [hunk 1]
```diff
@@ -40,4 +40,8 @@
 }
 
 echo 'Please use ./occ system:cron' . PHP_EOL;
-exit(1);
+$return = system('./occ system:cron');
```
Comment (PVince81) [inline cron.php]: this implies that PHP is allowed to call system functions

is there maybe another way where we instantiate it through Symfony like we do for unit tests ?

Comment (DeepDiver1975) [reply cron.php]: > this implies that PHP is allowed to call system functions

at this level where php scripts are called from crontab we can assume that we can call system functions.

Adding more complexity here by using symfony is an overkill from my perspective

Comment (PVince81) [reply cron.php]: okay, so assuming that the php.ini for the cli is more permissive

**Review by PVince81 — APPROVED** [review]
:+1:

**Comment by PVince81** [comment]
@micbar

File: cron.php (hunk 1 above)
Comment (micbar) [inline cron.php]: Coding standard is complaining

**Review by micbar — CHANGES_REQUESTED** [review]
Could you push the changes?

**Review by micbar — APPROVED** [review]

**Pull request merged**